inspect.cpp

```diff
@@ -899,6 +900,14 @@ namespace Sass {
     Compound_Selector*           head = c->head();
     Complex_Selector*            tail = c->tail();
     Complex_Selector::Combinator comb = c->combinator();
+
+    if (c->has_line_feed()) {
```

Comment:
This block appears to be the cause of the broken selector output reported in https://github.com/sass/libsass/pull/1249#issuecomment-120948335 and https://github.com/sass/libsass/pull/1249#issuecomment-120950668